### ISSUE TITLE
Can't create cluster. i/o timeout

### ISSUE THREAD

**Issue by da-source** [cortexlabs/cortex#1527]
I'm trying to spin up a cluster with `c6g.medium `instances but I'm getting this:

```
aws resource                               cost per hour
1 eks cluster                              $0.10
1 - 3 c6g.medium instances for your apis   $0.034 each
1 - 3 50gb ebs volumes for your apis       $0.007 each
1 t3.medium instance for the operator      $0.0416
1 20gb ebs volume for the operator         $0.003
2 network load balancers                   $0.0225 each

your cluster will cost $0.23 - $0.31 per hour based on cluster size

cortex will also create an s3 bucket (fifth-5835589a93) and a cloudwatch log group (cortex)

would you like to continue? (y/n): y

￮ creating a new s3 bucket: fifth-5835589a93 ✓
￮ using existing cloudwatch log group: cortex ✓
￮ creating cloudwatch dashboard: fifth ✓
￮ creating api gateway: fifth ✓
￮ spinning up the cluster ... (this will take about 15 minutes)

[ℹ]  eksctl version 0.27.0
[ℹ]  using region us-east-1
[ℹ]  subnets for us-east-1a - public:192.168.0.0/19 private:192.168.64.0/19
[ℹ]  subnets for us-east-1b - public:192.168.32.0/19 private:192.168.96.0/19
[ℹ]  nodegroup "ng-cortex-operator" will use "ami-07250434f8a7bc5f1" [AmazonLinux2/1.17]
[ℹ]  nodegroup "ng-cortex-worker-on-demand" will use "ami-07ee96b515734922b" [AmazonLinux2/1.17]
[ℹ]  using Kubernetes version 1.17
[ℹ]  creating EKS cluster "fifth" in "us-east-1" region with un-managed nodes
[ℹ]  2 nodegroups (ng-cortex-operator, ng-cortex-worker-on-demand) were included (based on the include/exclude rules)
[ℹ]  will create a CloudFormation stack for cluster itself and 2 nodegroup stack(s)
[ℹ]  will create a CloudFormation stack for cluster itself and 0 managed nodegroup stack(s)
[ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-1 --cluster=fifth'
[ℹ]  CloudWatch logging will not be enabled for cluster "fifth" in "us-east-1"
[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-1 --cluster=fifth'
[ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "fifth" in "us-east-1"
[ℹ]  2 sequential tasks: { create cluster control plane "fifth", 2 sequential sub-tasks: { 2 sequential sub-tasks: { tag cluster, update default addons }, 2 parallel sub-tasks: { create nodegroup "ng-cortex-operator", create nodegroup "ng-cortex-worker-on-demand" } } }
[ℹ]  building cluster stack "eksctl-fifth-cluster"
[ℹ]  deploying stack "eksctl-fifth-cluster"
[✔]  tagged EKS cluster (cortex.dev/cluster-name=fifth)
[!]  1 error(s) occurred and cluster hasn't been created properly, you may wish to check CloudFormation console
[ℹ]  to cleanup resources, run 'eksctl delete cluster --region=us-east-1 --name=fifth'
[✖]  getting list of API resources for raw REST client: Get "https://0360B79A70FF834E7A62116404B9622A.gr7.us-east-1.eks.amazonaws.com/api?timeout=32s": dial tcp 3.227.69.239:443: i/o timeout
Error: failed to create cluster "fifth"

please run `cortex cluster down` to delete the cluster before trying to create this cluster again
```

How can this be solved?

**Comment by deliahu** [comment]
@da-source When I tried to reproduce this with your configuration (c6g.medium instances in us-east-1), the cluster was able to spin up successfully. According to https://github.com/weaveworks/eksctl/issues/2201, this could be a transient issue with AWS.

That said, ARM instances aren't yet supported by Cortex, so you will run into issues down the road. I just created https://github.com/cortexlabs/cortex/issues/1528 to track support for ARM, and we'll add a validation check in our next release that will block ARM-based instances for now.

In the meantime, can you try again with a non-ARM instance type? Here's a [list of common instance types](https://docs.cortex.dev/cluster-management/ec2-instances). t3.small/t3a.small is probably the most similar to c6g.medium (and it's cheaper), or t3.medium/t3a.medium is good if you need a bit more memory.

**Comment by deliahu** [comment]
@da-source we released v0.22 today, which will throw an error during `cortex cluster up` if an ARM-based instance is requested. Thanks for bringing this to our attention!
